### DISCUSSION TITLE
Site does not show IP4 when go to certain sites

### DISCUSSION THREAD

**Discussion by Greytso** [angristan/openvpn-install#1104] [Help]
Hi all. Please tell me, I've already broken my head. When going to the site, bet365 does not show ip4, although everything is in order on the same browserleaks site. I'm attaching screenshots for your understanding. I use OpenVpn, the problem, as I understand it, is that the udp protocol does not completely transfer traffic to the browser. I would appreciate any help, thanks!
Here is the screenshot when switching to bet365
http://joxi.ru/LmGwGBQCBa4EMm

Here is the screenshot when switching to browserleaks
http://joxi.ru/V2VwE9kCBjEPGA

**Comment by TinCanTech** [comment]
@angristan Personally, I would delete this entire issue as **spam**.
